winkelmand.php

```diff
@@ -60,22 +69,54 @@
                                     </form>
                                 </div>
                             </h1>
-
                         </div>
-
-
-
+                        
                         <?php
                     }
                 }
                 else{
                     echo "Winkelmandje is leeg.";
                 }
-                ?>
+                if($totalprice > 0){
+                    $totalprice = str_replace('.', ',', sprintf("€%.2f", $totalprice));
+                    ?>
                 <div>
-                    <h1 class="StockItemPriceTextcart">Totaal prijs: <?php echo '€'.str_replace('.', ',', $totalprice); ?> </h1>
+                    <h1 class="StockItemPriceTextcart">Totaal prijs: <?php echo $totalprice ?> </h1>
                 </div>
+                <?php } ?>
+            </div>
         </ul>
+        <?php
+        if (isset($_COOKIE["basket"]) AND !cookieEmpty()) {
+            ?>
+        <div class="bonnetje-wrapper">
+            <?php
+
+                $basket_contents = json_decode($_COOKIE["basket"], true);
+                ?>
+            <table>
+                <th>Product</th>
+                <th>Aantal</th>
+                <th>Prijs</th>
+
+                <?php
+                foreach ($basket_contents as $item) {
+                    $StockItem = getStockItem($item["id"], $databaseConnection);
+                    echo ("<tr> <td>" . $StockItem['StockItemName'] . "</td>");
+                    echo ("<td>" . $item['amount'] . "</td>");
+                    echo "<td>".sprintf("€%.2f", $StockItem['SellPrice'] * $item["amount"]);
+                }
```

Comment:
Als er zoveel echo's zijn zal ik ervoor kiezen om dit uit de php tags te halen of er 1 echo of print van te maken op meerdere regels